internal/rootcoord/proxy_client_manager.go

```diff
@@ -207,3 +207,28 @@ func (p *proxyClientManager) UpdateCredentialCache(ctx context.Context, request
 	}
 	return group.Wait()
 }
+
+func (p *proxyClientManager) RefreshPolicyInfoCache(ctx context.Context, req *proxypb.RefreshPolicyInfoCacheRequest) (*commonpb.Status, error) {
+	p.lock.Lock()
+	defer p.lock.Unlock()
+	defer func() {
+		if err := recover(); err != nil {
```

Comment:
Why catch a panic exception here?  it will cause security risk if refresh policy fail